backend/src/main/java/ai/verta/modeldb/experimentRun/FutureExperimentRunServiceImpl.java

```diff
@@ -85,6 +85,10 @@ public void getExperimentRunsInProject(
                       futureExperimentRunDAO.findExperimentRuns(
                           FindExperimentRuns.newBuilder()
                               .setProjectId(request.getProjectId())
+                              .setPageLimit(request.getPageLimit())
+                              .setPageNumber(request.getPageNumber())
+                              .setAscending(request.getAscending())
+                              .setSortKey(request.getSortKey())
```

Comment:
missing fields to set in find when calling `getExperimentRunInProject` endpoint which is added now